egs/wsj/s5/steps/align_fmllr_lats.sh

```diff
@@ -54,7 +54,7 @@ sdata=$data/split$nj
 
 mkdir -p $dir/log
 echo $nj > $dir/num_jobs
-[[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh $data $nj || exit 1;
+[[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh --allow-uneven-split $data $nj || exit 1;
```

Comment:
After thinking about this more, I think it will cause too many problems to existing recipes if we make it an error to have uneven splits.  Best to have it just be a warning.  (Should go to stderr, obviously: print STDERR "string...";)

Reply:
Sorry, I didn't quite get what you are asking. Do you mean instead of the `die` [here](https://github.com/kaldi-asr/kaldi/blob/6d06c1bedcc90802a221bbae507799391beccf2e/egs/wsj/s5/utils/split_scp.pl#L209), there should be a Warning printed to STDERR instead?